tmt/steps/prepare/install.py

```diff
@@ -165,17 +165,25 @@ def go(self, guest):
         packages = self.get('package', [])
         repo_packages = []
         local_packages = []
+        repo_debuginfos = []
         directories = self.get('directory', [])
         if not packages and not directories:
             self.debug("No packages for installation found.", level=3)
             return
 
         # Detect local packages and directories
+        # and split debug{info,source} if installing from repo
         for package in packages:
             if package.endswith('.rpm'):
                 local_packages.append(package)
             else:
-                repo_packages.append(package)
+                if 'debuginfo' in package or 'debugsource' in package:
```

Comment:
Would be probably safer to use `re` or `str.endswith()` and match end of line here, quick check shows there might be packages with `debuginfo` in the middle of the name such as `elfutils-debuginfod-client`.

Reply:
That exactly what I was thinking about. debugifno indeed isn't always in the end. My example was perl-debuginfo.i686 on x86_64 system.

Reply:
I'll leave it as is. Although str.endswith would be safe, this approach gives is additional bit of flexibility.